### ISSUE TITLE
Issue with missing live streams on TikTok Web

### ISSUE THREAD

**Issue by FabLeKebab** [Coombaa/TTAutoRecord#51]
Hello,

On TikTok Web, I’ve noticed that some of the live streams from the people I follow are not showing up, even though they are visible on the mobile app. I’ve just reported the issue to TikTok.

Is anyone else experiencing this problem?

**Comment by Luly-bit** [comment]
Oh, I thought it was just me, but apparently not. Basically, the issue only happens with new subscriptions. The old ones work fine, but the new ones don’t update on my web profile. I’m using the same account, and the subscriptions are synced correctly (the number is the same on the TikTok app and the web version). However, the new live streams don’t show up on the web version, no matter the browser or even if I use a different computer...

**Comment by FabLeKebab** [comment]
The solution I found was to re-follow the accounts I’m subscribed to. I did it, and it works perfectly! Sure, it took me some time, but at least my account is clean and well-organized. I’m going to do the same on another account to have a second one, just in case, and maybe more later. What’s strange is that I haven’t been banned from my main account, but whatever. If this happens to others, I think this method is the simplest and most effective!  

Closing the issue.
